operator/src/test/java/org/keycloak/operator/testsuite/integration/UpdateTest.java

```diff
@@ -57,6 +59,7 @@ public class UpdateTest extends BaseOperatorTest {
     @ParameterizedTest(name = "testImageChange-{0}")
     @EnumSource(UpdateStrategy.class)
     public void testImageChange(UpdateStrategy updateStrategy) throws InterruptedException {
+        Assumptions.assumeTrue(operatorDeployment != OperatorDeployment.local_apiserver || updateStrategy != UpdateStrategy.AUTO);
```

Comment:
Nitpick: To unify the approach, we could add the `@DisabledIfApiServerTest` annotation here and have the update strategy in a separate assumption. WDYT?

Reply:
There are 3 tests being run here. We can always run 2 of them. It's only the AUTO mode that is the problem with local_apiserver. Adding DisabledIfApiServerTest will cause the other tests not to run.